css/style.css

```diff
@@ -0,0 +1,511 @@
+  top: 10px;
+  right: 10px;
+}
+
+a.btn-one {
+  text-decoration: none;
+  color: #4053fc;
+}
+
+a.btn-two {
+  text-decoration: none;
+}
+
+.btn-tags {
+  margin-bottom: 30px;
+}
+
+.section-about {
+  background-color: white;
+  border-top-right-radius: 100px;
+}
+
+.section-about h1 {
+  position: static;
+  width: 920px;
+  height: 52px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 35px;
+  letter-spacing: 0.37px;
+  color: #091e42;
+  margin: 40px 10px 30px 0;
+  font-size: 35px;
+}
+
+.about-grid {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(420px, 1fr));
+  grid-gap: 20px;
+  align-items: center;
+}
+
+.about-grid > article {
+  padding-bottom: 50px;
+}
+
+.contact-form {
+  background-color: #fff;
+}
+
+.contacts-form {
+  background: url("../image/contactbackground.png");
+  background-size: 60% 100%;
+  background-repeat: no-repeat;
+  background-position: top right;
+  background-color: #6070ff;
+  font-family: "Poppins", sans-serif;
+  border-top-left-radius: 100px;
+  width: 100%;
+  padding: 13px 24px 0;
+  box-sizing: border-box;
+}
+
+.contacts-form h1:first-child {
+  text-align: center;
+  font-weight: bold;
+  font-size: 2.5rem;
+  margin-bottom: 0;
+  color: #fff;
+}
+
+.contacts-form h1:first-of-type {
+  color: #ebebff;
+  font-size: 2.5rem;
+  text-align: center;
+  font-weight: bold;
+  margin-top: 80px;
+}
+
+.contacts-form p:nth-child(2) {
+  color: #fff;
+  text-align: center;
+  font-size: 1.25rem;
+  margin-bottom: 40px;
+}
+
+.email {
+  width: 100%;
+  height: 48px;
+  margin-bottom: 20px;
+  border-radius: 8px;
+  border: none;
+  padding: 15px;
+  font-size: 1.25rem;
+  color: #172b4d;
+  box-sizing: border-box;
+}
+
+.msg {
+  width: 100%;
+  padding: 16px;
+  box-sizing: border-box;
+  border-radius: 8px;
+  margin-bottom: 22px;
+  color: black;
+  font-family: Poppins, sans-serif;
+  font-size: 1.25rem;
+}
+
+.email:focus,
+.message:focus {
+  border: none;
+  outline: none;
+}
+
+.name {
+  width: 100%;
+  height: 48px;
+  margin-bottom: 20px;
+  border-radius: 8px;
+  border: none;
+  padding: 15px;
+  font-size: 17px;
+  color: #172b4d;
+  box-sizing: border-box;
+}
+
+.send {
+  color: #6070ff;
+  font-size: 1.06rem;
+  border-radius: 8px;
+  margin-bottom: 32px;
+  border: none;
+  padding: 14px;
+}
+
+.send:hover {
+  border: 1px solid white;
+}
+
+@media only screen and (max-width: 769px) {
+  /* for mobile phone */
+  .navbar {
+    flex-direction: column;
+  }
+
+  .nav-list {
+    list-style-type: none;
+    width: 100%;
+    text-align: center;
+    padding-top: 15px;
+    display: none;
+  }
+
+  .nav-list .list-item {
+    display: block;
+    padding: 10px;
+    border-top: 1px solid white;
+    color: black;
+  }
+
+  .navbar a {
+    color: #5571fe;
+    text-decoration: none;
+    font-size: 20px;
+  }
+}
+
+.menu {
+  display: block;
+  position: absolute;
+  right: 15px;
+  top: 15px;
+}
+
+@media only screen and (min-width: 768px) {
```

Comment:
- [ ] Uses media queries for the Desktop breakpoint in the CSS file. (the value for min-width property should be in range of 992px - 1100px). Please fix this. 🌵